composer.json

```diff
@@ -1,25 +1,192 @@
 {
     "name": "ymcatwincities/openy",
-    "description": "Open Y distribution",
     "type": "drupal-profile",
+    "description": "Open Y distribution",
```

Comment:
could you pull upstream code to this PR?

Reply:
The branch has been updated